src/core/js/router.js

```diff
@@ -130,24 +130,21 @@ define([
 
     async handleId(id) {
       const rootModel = Adapt.router.rootModel;
-      const model = (!id) ? rootModel : Adapt.findById(id);
+      let model = (!id) ? rootModel : Adapt.findById(id);
 
       if (!model) {
         // Bad id
         this.model.set('_canNavigate', true, { pluginName: 'adapt' });
         return;
       }
 
-      id = model.get('_id');
-
+      // Keep the routed id incase it needs to be scrolled to later
       const isContentObject = (model instanceof ContentObjectModel);
-      if (!isContentObject) {
-        // Allow navigation.
-        this.model.set('_canNavigate', true, { pluginName: 'adapt' });
-        // Scroll to element
-        Adapt.navigateToElement('.' + id, { replace: true, duration: 400 });
-        return;
-      }
+      const navigateToId = model.get('_id');
```

Comment:
This variable isn't needed. We can just use `id`

Reply:
This is the original id from the URL, so say `c-25`.

Line 147 will change the `id` variable to the content object for rendering.

Line 202 needs the original id to perform the scrollTo.

Reply:
ah yes, missed the new model setting 🙈